bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -914,6 +914,15 @@ impl Connection {
         }?;
         Ok(self)
     }
+
+    /// Allows the quic library to check if session tickets are expected
+    pub fn client_resumption_enabled(&self) -> bool {
```

Comment:
Just to make sure I understand: there won't be an s2n-quic API to enable session resumption that calls `s2n_config_set_session_ticket_cb()`, right? So to configure session resumption in s2n-quic the user sets a session ticket callback on an s2n_config and gives that to s2n-quic, so this API is needed in order for s2n-quic to know that the user did that?

Reply:
Yes, there are no new s2n-quic APIs for the session resumption feature, which means that s2n-quic currently doesn't know if a client enabled resumption or not. And that would be fine, except that s2n-quic has a memory optimization which throws out the TLS session when it isn't needed, so this API is needed to tell s2n-quic that it needs to keep the session around longer.

Reply:
Got it okay, thanks!

Since this is a connection API rather than a config API, would it make sense to first check that this is a client connection before returning true? Or is this not really a problem since it would be weird to try and call this on a server connection?

Reply:
Honestly I have a really hard time picturing someone using this on a server endpoint. Like, I think the name makes it clear where you are supposed to use it. LMK if you disagree though and I'll add it.